packages/checkout-ui-extensions/src/components/Form/Form.ts

```diff
@@ -25,14 +25,6 @@ export interface FormProps {
 }
 
 /**
- * The `Form` component should be used to wrap one or more form controls.
- * This component provides an "implicit submit" behavior, where buyers can
- * submit the form from any input by pressing "enter" on their keyboards. This
- * behavior is widely expected, and should be respected as often as possible.
- *
- * Unlike an HTML `form` element, this component does not support configuring
- * the descendant fields to be submitted via HTTP automatically. Instead, you
- * must provide an `onSubmit` callback that will perform the necessary HTTP
- * requests in JavaScript.
```

Comment:
> Why did we remove so much useful context? Having READMEs is nice, but this content was shown directly in the developer's editor when they are building an extension. We should keep it.

Agreed. It's just moved into the description and best practices:

https://github.com/Shopify/ui-extensions/pull/240/files#diff-bb181e6a7d1a24b1cf338924327e55c88165beccc847383fca23e5fa84adb7faR28

https://github.com/Shopify/ui-extensions/pull/240/files#diff-f5105ac91472e389c42d9ca882b407a78062647f355af8b2e0c0dd2b2832e78bR5

Reply:
> It would also be nice if we kept a reasonable line length, not super fun to review lines that get as long as the one below.

👍🏻 I'll fix up some of these long lines at the source

Reply:
To me, it is a big step backward to move content from these doc blocks (which are shown when you import components from `@shopify/checkout-ui-extensions`) to a README that is only shown if you navigate all the way into it on the dev docs site/ in this repo. I'm all for adding dedicated best practices docs, but all of the important information for using a component should be in these doc blocks too.

Reply:
That makes sense. I've restored this content.